views/about_me.erb

```diff
@@ -0,0 +1,28 @@
+<html>
+<head>
+  <title>About Me</title>
+</head>
+<header>
+  <h3><a href="index.html" />Home</a>   <a href="blog.html" />Blog</a>   <a href= "projects.html" />Projects</a></h3>
```

Comment:
Do you mean like this?
<h3>
      <a href="about_me.html" />About Me</a>
      <a href= "projects.html" />Projects</a>
      <a href="blog.html" />Blog</a>
    </h3>